curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63b60821c855d01b1eda3c0b.md

```diff
@@ -7,8 +7,7 @@ dashedName: step-5
 
 # --description--
 
-Now create a `fieldset` with an `id` set to `lunch`, and a corresponding `legend` and `div` element.
-
+Now create a `fieldset` with an `id` set to `lunch`. Inside this `fieldset`, add a `legend` element with the text content `lunch`, and a `div` element with a `class` set to `input-container`.
```

Comment:
Please use the solution my colleague @jdwilkin4 provided on the issue discussion, also put some space between the `description` and `hint` tags. Thank you for contributing 🙏

```suggestion
Next, create a `fieldset` element with the `id` set to `lunch`.

Within that element, create a `legend` element with the text `Lunch`, and an empty `div` with the `class` set to `input-container`.
```